### PR TITLE
refactor(UI): Privacy Note

### PR DESCRIPTION
**Pull request by amanueltsfy**
image

## Issues
- #38 
- Automatically add URL scheme if it's missing. Refer to https://github.com/Focus-Bear/windows-app-v2/issues/42#issuecomment-1441190853

### CODE REVIEW

File: focus-blocked/script.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const focus_tip_old_url = `<div class='notice-wrapper'>
                     ${
                       block_type
                         ? ''
-                        : `<a href=${current_url}>Click here to re-open the original URL ${
+                        : `<a href=${`https://${current_url}`}>Click here to re-open the original URL ${
```
Comment (jeznag) [inline focus-blocked/script.js:27]: I don't think this is the right approach. https:// is already there in some cases. You need to process old_url to add https:// if it is not set

File: focus-blocked/script.js [hunk 2]
```diff
@@ -24,7 +25,7 @@ const focus_tip_old_url = `<div class='notice-wrapper'>
                     ${
                       block_type
                         ? ''
-                        : `<a href=${current_url}>Click here to re-open the original URL ${
+                        : `<a href=${`${current_url}`}>Click here to re-open the original URL ${
```
Comment (jeznag) [inline focus-blocked/script.js:28]: template string not necessary

File: focus-blocked/script.js (hunk 1 above)
Comment (amanueltsfy) [reply focus-blocked/script.js]: image

File: focus-blocked/script.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 const urlParams = new URLSearchParams(window.location.search);
-const current_url = urlParams.get('old_url');
+let current_url = urlParams.get('old_url');
+current_url = current_url.startsWith('https') ? current_url : `https://${current_url}`
```
Comment (jeznag) [inline focus-blocked/script.js:3]: Should be starts with http

Otherwise you'll make it http://https://

**Pull request merged**